packages/camera/camera_avfoundation/example/ios/RunnerTests/FLTCamSampleBufferTests.m

```diff
@@ -11,31 +11,98 @@
 
 /// Includes test cases related to sample buffer handling for FLTCam class.
 @interface FLTCamSampleBufferTests : XCTestCase
-
+@property(readonly, nonatomic) dispatch_queue_t captureSessionQueue;
+@property(readonly, nonatomic) FLTCam *camera;
+@property(readonly, nonatomic) CMSampleBufferRef sampleBuffer;
```

Comment:
let's avoid having a shared state across tests.

Reply:
`setUp` and `tearDown` are called before and after each individual test (function) so I do not understand how this is a shared state. This same thing is used also in `StreamingTest.m`. I also suppose there are going to be additional tests which will require a similar setup in near future.